packages/instrumentation-aws-sdk/test/load-instrumentation.ts

```diff
@@ -30,6 +30,11 @@ import {
 } from '@opentelemetry/sdk-metrics';
 import { AwsInstrumentation } from '../src';
 
+// This is a meagre testing of just a single value of
+// OTEL_SEMCONV_STABILITY_OPT_IN, because testing multiple configurations of
+// `AwsInstrumentation` in this all-in-one-process test setup is difficult.
+process.env.OTEL_SEMCONV_STABILITY_OPT_IN = 'http/dup';
+
```

Comment:
Reviewer note: I realize I am not testing all the values of `OTEL_SEMCONV_STABILITY_OPT_IN` in the tests here. My plea is that it is not worth the effort:

- Currently the tests run all in one process, and all the test cases share one instance of AwsInstrumentation (from "test/load-instrumentation.ts"). Loading a second instrumentation (to test separate configurations of it) is generally fraught.
- This instrumentation makes very minimal use of the `...OPT_IN` envvar: just for a single HTTP-related attribute.
- After ~6 months we will consider the HTTP migration to be complete and will be able to remove this `...OPT_IN` handling.

Reply:
I see you chose the option to send both, so should be good enough to check if both ways are being generated, but you didn't update the test to confirm the stable values are being sent, like you did on the other file. Is there nothing here to check to confirm the stable value is also being send?

Reply:
Commit 71a26a20.
Good call. I had missed one place.